lighthouse-core/lib/log.js

```diff
@@ -113,23 +115,23 @@ class Log {
     Log._logToStdErr(`${prefix}:${level || ''}`, [data.method, snippet]);
   }
 
-  static log(title) {
-    Log.events.issueStatus(title, arguments);
-    return Log._logToStdErr(title, Array.from(arguments).slice(1));
+  static log(title, ...args) {
+    Log.events.issueStatus(title, ...args);
+    return Log._logToStdErr(title, args);
```

Comment:
> `_logToStdErr` doesn't seem to have been changed at all did it already support this?

Maybe I'm misunderstanding you, but isn't this right? `args` is an array of all but the first argument, equivalent to the old `Array.from(arguments).slice(1))`?